swagger_to_sdk_config.json

```diff
@@ -2,7 +2,6 @@
   "$schema": "https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/schemas/swagger_to_sdk_config.schema.json",
   "meta": {
     "autorest_options": {
-      "version": "preview",
```

Comment:
For now this would need to be 2.0.4413, or 'previous',  until the polling for ttags is fixed.  Would also need a similar fix in the AzureSDKAutomation fork of the repo

Reply:
`version:V2` might be the safest bet. 

That at least won't float arbitrarily.

Reply:
Moved to "V2" per @fearthecowboy suggestion

![image](https://user-images.githubusercontent.com/1050156/75378228-2ce19800-5888-11ea-8c99-cd787cd67438.png)